gdpr/impl.go

```diff
@@ -10,51 +10,51 @@ import (
 	tcf2ConsentConstants "github.com/prebid/go-gdpr/consentconstants/tcf2"
 	"github.com/prebid/go-gdpr/vendorconsent"
 	tcf2 "github.com/prebid/go-gdpr/vendorconsent/tcf2"
-	"github.com/prebid/go-gdpr/vendorlist"
 	"github.com/prebid/prebid-server/config"
 	"github.com/prebid/prebid-server/openrtb_ext"
 )
 
 type permissionsImpl struct {
-	cfg              config.GDPR
-	gdprDefaultValue Signal
-	purposeConfigs   map[consentconstants.Purpose]config.TCF2Purpose
-	vendorIDs        map[openrtb_ext.BidderName]uint16
-	fetchVendorList  map[uint8]func(ctx context.Context, id uint16) (vendorlist.VendorList, error)
+	fetchVendorList       VendorListFetcher
+	gdprDefaultValue      string
+	hostVendorID          int
+	nonStandardPublishers map[string]struct{}
+	cfg                   TCF2ConfigReader
+	vendorIDs             map[openrtb_ext.BidderName]uint16
 }
 
 func (p *permissionsImpl) HostCookiesAllowed(ctx context.Context, gdprSignal Signal, consent string) (bool, error) {
-	gdprSignal = SignalNormalize(gdprSignal, p.cfg)
+	gdprSignal = SignalNormalize(gdprSignal, config.GDPR{DefaultValue: p.gdprDefaultValue})
```

Comment:
Instead of creating my own `SignalNormalize` that takes either a GDPR signal value or its string equivalent, I figured I would just wrap the default value specified when the permissions object was created in an instance of `config.GDPR` and pass that in. With this new implementation, `config.GDPR` is no longer available on the object.